app/components/card-sort/template.hbs

```diff
@@ -0,0 +1,54 @@
+<p><b>Describe the situation.</b> Please use these items to describe the situation you experienced
+    yesterday. There are 90 items, which will appear one at a time. Begin by placing each item into one of
+    three boxes. Use the "Characteristic" box on the right for items that accurately describe the situation; use
+    the "Uncharacteristic" box on the left for items that are undescriptive of the situation, and use the "Neutral"
+    box for items that are irrelevant, unclear, or about which you are uncertain. When you are finished, press
+    "Continue."
+</p>
+<br>
+<div style="height: 75px; overflow: hidden; background-color:#f5f5f5; padding: 10px">
```

Comment:
Can you please move these styles to a separate .css or .scss file? 

Take a look how the styles are structured here: https://github.com/CenterForOpenScience/experimenter/tree/develop/app/styles